packages/ag-charts-enterprise/src/components/popover/popover.css

```diff
@@ -1,10 +1,12 @@
 .ag-charts-popover {
+    --popover-layer: 0;
+
     position: absolute;
     border: var(--ag-charts-popover-border);
     border-radius: var(--ag-charts-popover-border-radius);
     background: var(--ag-charts-popover-background-color);
     color: var(--ag-charts-popover-foreground-color);
     font-family: var(--ag-charts-font-family);
     font-size: var(--ag-charts-popover-font-size);
-    z-index: var(--ag-charts-layer-ui-overlay);
+    z-index: calc(var(--ag-charts-layer-ui-overlay) + var(--popover-layer));
```

Comment:
If we allow providing a container to attach to the popover so the popover is a child of the dialog, we can avoid this z-index stuff